### ISSUE TITLE
FNaFb1 - Various Generation Failures

### ISSUE THREAD

**Issue by WolfBoi008** [WolfBoi008/Five-Nights-at-Fuckboy-s-Complete-Collection#16]
FNaFb1 will not generate despite my best efforts. If anyone can help with Manual functions/hooks, contact me.

**Comment by BonkHolmes** [comment]
Current issue is an accessibility failure for everything with logic.

**Comment by BonkHolmes** [comment]
This has now been fixed
